### PR TITLE
fix to be consistent with listing below

### PR DESCRIPTION
**Pull request by michalkoza**
The listing just below includes `/` as well so I figure the language should be `three` here

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @michalkoza,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: akka-docs/src/main/paradox/typed/guide/tutorial_1.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ As we covered in the @scala[[Quickstart Guide](https://developer.lightbend.com/g
 
 ![actor tree diagram](diagrams/actor_top_tree.png)
 
-In fact, before your first actor is started, Akka has already created two actors in the system. The names of these built-in actors contain _guardian_. The guardian actors include:
+In fact, before your first actor is started, Akka has already created three actors in the system. The names of these built-in actors contain _guardian_. The guardian actors include:
```
Comment (johanandren) [inline akka-docs/src/main/paradox/typed/guide/tutorial_1.md:38]: With typed, the user actor "your first" is the user guardian, so the two that are started before it is the root guardian and the system guardian, so two here is correct.

Comment (michalkoza) [reply akka-docs/src/main/paradox/typed/guide/tutorial_1.md]: gotcha! how about rephrasing it then like this?

**Review by johanandren — APPROVED** [review]
Looks like an improvement, thanks!

**Pull request merged**